common/thrift_router.h

```diff
@@ -410,28 +464,33 @@ class ThriftRouter {
       return socket->good();
     }
 
-    void createOrFixClientsFor(const std::set<const Host*>& hosts) {
-      for (auto host : hosts) {
-        auto itor = clients_->find(host->addr);
-        if (itor != clients_->end() &&
-            is_client_good(itor->second.client.get())) {
-            // has the client and it is good
-          continue;
-        }
+    bool createOrFixClientFor(const Host* host) {
```

Comment:
maybe it would be helpful to add some method description here on what it returns.